driver/server/server.go

```diff
@@ -2,16 +2,21 @@ package server
 
 import (
 	"flag"
+	"io"
 	"net"
 	"os"
 
+	jaegercfg "github.com/uber/jaeger-client-go/config"
+	"google.golang.org/grpc"
+
 	cmdutil "gopkg.in/bblfsh/sdk.v2/cmd"
 	"gopkg.in/bblfsh/sdk.v2/driver"
 	"gopkg.in/src-d/go-errors.v1"
 )
 
 var (
 	ErrInvalidLogger       = errors.NewKind("invalid logger configuration")
+	ErrInvalidTracer       = errors.NewKind("invalid tracer configuration")
```

Comment:
I would prefer adding go doc at least for the new code we are adding.

Reply:
Agreed. E.g., `// ErrInvalidTracer is reported by blah when an invalid trace configuration is found`
or whatever is actually true here.